wire/core/Pageimage.php

```diff
@@ -1810,6 +1897,120 @@ protected function ___install($filename) {
+		$info['iptcRaw'] = $osInfo['iptcRaw'];
+		
+		// WEBP
+		$info['WEBP COPY'] = array();
+		if($this->hasWebp()) {
+			$info['WEBP COPY']['hasWebp'] = true;
+			$info['WEBP COPY']['webpUrl'] = $this->webpUrl;
+			if(isset($finalOptions['webpQuality'])) $info['WEBP COPY']['webpQuality'] = $finalOptions['webpQuality'];
+			$info['WEBP COPY']['filesize'] = filesize($this->webpFilename());
+			$info['WEBP COPY']['savings in percent'] = 100 - intval($info['WEBP COPY']['filesize'] / ($info['filesize'] / 100));
+		} else {
+			$info['WEBP COPY']['hasWebp'] = false;
+		}
+		// VARIATIONS
+		if($depth < 2) {
+			$info['VARIATIONS'] = array();
+			$variations = $this->getVariations(array('info' => true, 'verbose' => false));
+			foreach($variations as $name) {
+				$info['VARIATIONS'][] = $name;
+			}
+			#if(empty($info['VARIATIONS'])) unset($info['variations']);
+		}
+		$depth--;
+		// ENGINES
+		$info['ENGINE(S)'] = array();
+		$info['ENGINE(S)']['neededEngineSupport'] = strtoupper($oSizer->getImageInfo());
+		$a = [];
+		$modules = $this->wire('modules');
+		$engines = array_merge($oSizer->getEngines(), array('ImageSizerEngineGD'));
+		foreach($engines as $moduleName) {
+			$configData = $modules->getModuleConfigData($moduleName);
+			$priority = isset($configData['enginePriority']) ? (int) $configData['enginePriority'] : 0;
+			$a[$moduleName] = "priority {$priority}";
+		}
+		asort($a, SORT_STRING);
+		$info['ENGINE(S)']['installedEngines'] = $a;
+		unset($a, $moduleName, $configData, $engines, $priority, $engines, $modules);
+		$info['ENGINE(S)']['selectedEngine'] = $oSizer->getEngine()->className;
+		// OPTIONS
+		$info['OPTIONS HIERARCHY'] = array();
+		$info['OPTIONS HIERARCHY']['imageSizerOptions'] = $this->wire('config')->imageSizerOptions;
+		$info['OPTIONS HIERARCHY']['individualOptions'] = $options;
+		$info['OPTIONS HIERARCHY']['finalOptions'] = $finalOptions;
+		
+		if(!$returnAsString) {
+			return $info;  // return as array
+		}
+		
+		// make a beautyfied var_dump
+		ob_start();
+		var_dump($info);
+		$content = ob_get_contents();
+		ob_end_clean();
+		$m = 0;
+		preg_match_all('#^(.*)=>#mU', $content, $stack);
+		$lines = $stack[1];
+		$indents = array_map('strlen', $lines);
+		if($indents) $m = max($indents) + 1;
+		$content = preg_replace_callback(
+			'#^(.*)=>\\n\s+(\S)#Um',
+			function($match) use ($m) {
+				return $match[1] . str_repeat(' ', ($m - strlen($match[1]) > 1 ? $m - strlen($match[1]) : 1)) . $match[2];
+			},
+			$content
+		);
+		$content = preg_replace('#^((\s*).*){$#m', "\\1\n\\2{", $content);
+		$content = str_replace(array('<pre>', '</pre>'), '', $content);
+		if(isset($_SERVER['HTTP_HOST'])) {
+			// build output for HTML
+			$return = "<pre style=\"margin:10px 10px 10px; padding:10px 10px 10px 10px; background-color:#F2F2F2; color:#000; border:1px solid #333; font-family:'Hack', 'Source Code Pro', 'Lucida Console', 'Courier', monospace; font-size:12px; line-height:15px; overflow:auto;\">{$content}</pre>";
```

Comment:
Minor note on this one: `10px 10px 10px` and `10px 10px 10px 10px` are both identical to simple `10px` :)